setup_undercloud.yml

```diff
@@ -30,24 +31,36 @@
       delay: 10
       with_items: "{{ async_install }}"
 
+    - include_tasks: tasks/get_interpreter.yml
+      vars:
+        hostname: "{{ undercloud_hostname }}"
+        user: "root"
+      ignore_errors: yes
+    - include_tasks: tasks/update_repo.yml
```

Comment:
please pin this task to be executed only for  osp13 and osp16.2

Reply:
Updated